src/test_loader.rs

```diff
@@ -287,6 +287,7 @@ fn get_line_addresses<'data>(
                         config.include_tests() || !k.path.starts_with(project.join("tests"))
                     })
                     .filter(|(ref k, _)| !(config.exclude_path(&k.path)))
+                    .filter(|(ref k, _)| (config.include_path(&k.path)))
```

Comment:
More redundant brackets

```suggestion
                    .filter(|(ref k, _)| config.include_path(&k.path))
```